app/models/video.rb

```diff
@@ -10,4 +12,16 @@ def self.search_by_title(search_string)
       where("LOWER(title) LIKE ?", "%#{search_string.downcase}%").order('created_at DESC')
     end  
   end  
+
+  def average_rating
+    if self.reviews.exists?
```

Comment:
you can also do `reviews.average(:rating).round(1) if reviews.average(:rating)`. See [here](http://guides.rubyonrails.org/active_record_querying.html#average)